zip-1337.md

```diff
@@ -0,0 +1,156 @@
+
+The crux of this proposal still centers around the idea that the Zcash Foundation and ECC share responsibility for protocol development, and assumes an eventual 2-of-2 agreement on the trademark and that both continue developing consensus-compatible node software that interacts with the Zcash network. But it mandates accountability for disbursement of funds to the ECC/Zcash Foundation, and expands outside stakeholder input on funds that _wouldn't_ be earmarked for the ECC/Zcash Foundation (similar to Placeholder's earlier version of their proposal), while it doesn’t preclude the possibility of migrating to broader “2-of-3” later on.
+
+### Why a trust?
+
+The main reason: reducing complexity creep in consensus code so the ECC and Zcash Foundation can budget for other features in NU4. Rather than try to incorporate some complex mechanism for dev fund disbursements on-chain, we can meet the NU4 deadline with the simplest possible code-change and spend more time ironing out the details of the trust "off-chain." Since both the ECC and the Zcash Foundation are based in the US, using a trust with well-specified criteria for disbursements is a reasonable path. This also fits in nicely with [lex-node's proposal](https://forum.zcashcommunity.com/t/dev-fund-supplemental-proposal-enforce-devfund-commitments-with-legal-charter) for legal covenants on funding.
+
+Security and Privacy Considerations
+===================================
+
+The biggest issue is custody of the funds under the trust's control, but I suspect this can be managed with a partnership with a custody partner. There's also the issue that non-public information would need to be verified and validated by the trust, but I view this as a net positive for the community. ("transparency for organizations, privacy for individuals")
+
+Reference implementation
+========================
+
+TBD, but it should be relatively simple to code in both zebra and zcashd.
+
+Issues and further discussion
+=============================
+
+- What are the tax implications for setting up the trust?
+- Who bears the cost of running the trust? (I think jointly split between the ECC and ZF makes sense)
+- Should the trust own the Zcash trademark? It could effectively function as a 2-of-2 agreement between the ECC and ZF.
+- Is there any way the shielded adoption metric could be gamed?
+- Are the amounts reasonable? Should the dev fund be less than 20% in aggregate?
+- How do we select representatives for the ZF Grants 3-of-5 body that aren't from the ECC or ZF?
```

Comment:
I think the community advisory panel model worked quite well for electing the ZF board. I would advocate this model, and perhaps expand it to include coin-voting and hashpower-voting in the future if there's a general sentiment that those groups are under-represented.